research/2022-08-20-introducing-waku-product.md

```diff
@@ -0,0 +1,110 @@
+Outreach, marketing, documentation must not be limited to the web, but target all platforms, from mobile to desktop to the cloud to the web.
+
+# Waku Product
+
+The Waku Product project is born from the need of having a consolidated effort to make the various Waku implementations **usable and used**.
+
+We are retiring the _Waku Connect_ branding in favour of the _Waku_ branding.
+When promoting, documenting or discussing Waku with Web3 projects,
+we are able to leverage all Waku implementations to better serve the user's needs:
+
+- Running a node for your projects and want to use Waku? Use [nwaku](https://github.com/status-im/nwaku).
+- Going mobile? Use [Waku React Native](https://github.com/status-im/waku-react-native).
+- C++ Desktop Game? Use [go-waku's C-Bindings](https://github.com/status-im/go-waku/tree/master/examples/c-bindings).
+- And of course, web app? Use [js-waku](https://github.com/status-im/js-waku).
+
+We are also consolidating the documentation for all implementations on a single website (work in progress)
+so it is easier for a developer to find the right information.
+
+We call the task of pushing for Waku to be used by other projects _platform outreach_.
+
+This year, we will also start the _operator outreach_ effort to push for users to run their own Waku nodes.
+Nwaku's documentation, stability and performance has improved,
+it is now easier to run your [own Waku node](https://github.com/status-im/nwaku/tree/master/docs/operators).
+
+Today, operators wannabes are most likely running their own nodes to support or use the Waku network.
+
+As we are [dogfooding Waku RLN](https://github.com/status-im/nwaku/issues/827), our novel economic spam protection protocol
+and looking at [incentivizating the Waku Store protocol](https://github.com/vacp2p/research/issues/99),
+we are adding further reasons to run your own Waku node.
+
+# In Conclusion
+
+We are committed to make Waku, the communication layer of Web3, a success.
+For this endeavour, the Vac program is focusing on inventing and improving the Waku protocols
+(among other [topics](https://github.com/vacp2p/research/issues/112)).
+And now, the Waku Product project is focusing on stabilizing, promoting, documenting Waku Software and supporting Waku users.
+
+What you can expect next:
+
+- [Scalability and performance studies](https://forum.vac.dev/t/waku-v2-scalability-studies/142/9) and improvement across Waku software,
+- [New websites](https://github.com/waku-org/waku.org/issues/15) to easily find documentation about Waku and its implementations,
+- Delivery of the latest Waku protocols in all code bases and cross client PoCs
+  ([noise](https://rfc.vac.dev/spec/35/), [noise-sessions](https://rfc.vac.dev/spec/37/),
+  [waku-rln-relay](https://rfc.vac.dev/spec/17/), etc),
+- Easier to [run your own waku node](https://github.com/status-im/nwaku/issues/828),
+- Improvement of existing protocols based on field-testing feedback and functional requirements
```

Comment:
Perhaps useful to make it explicit here that a large focus here is on restricted devices, e.g. browsers or limited-resource devices? Many people following Waku knows that we network capacity challenges (manifesting as unreliable message delivery) and this is a major part of the work to improve Waku capacity. I'd like the community to see visible progress, so they stay on board.